AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -3520,6 +3520,7 @@ ticketea.com#$#html { overflow:visible!important; }
 fantrax.com##toast > section.toast.ng-trigger-enterLeave
 tommy.com#$#.ReactModalPortal { display:none!important; }
 tommy.com#$#body { overflow:visible!important; }
+tommy.com#$#.overlay--cookies { display: none !important; }
```

Comment:
Why CSS? Scroll is not blocked. Old rules are obsolete, I guess.

Reply:
There already exists
`ar.tommy.com#@$#body { overflow:visible!important; }`